code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -675,6 +675,14 @@
 	if(hypo.penetrates)
 		to_chat(user, span_notice("[hypo] already has a piercing mechanism!"))
 		return FALSE
+
+/obj/item/reagent_containers/autoinjector/phytosiansyndicateshit
```

Comment:
I do not know how you did it but you managed to put this square in the middle of a proc